PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/WebAppRunState.java

```diff
@@ -221,9 +230,16 @@ private void uploadWarArtifact(@NotNull final FileInputStream input, @NotNull fi
         }
     }
 
+    private void ensureWebAppsFolderExist(@NotNull FTPClient ftp) throws IOException {
+        ftp.getStatus(WEB_APP_BASE_PATH);
```

Comment:
`public String getStatus(String pathname) throws IOException`  issue the FTP STAT command to the server for a given pathname. This should produce a listing of the file or directory.
`STAT` command:
https://support.solarwinds.com/Success_Center/Serv-U_Managed_File_Transfer_Serv-U_FTP_Server/Knowledgebase_Articles/STAT_FTP_command